### PR TITLE
Add package versions in requirements.txt

### PR DESCRIPTION
**Pull request by turbatuandrei**
Package versions for direct dependecies.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-datetime
-altair
-altair_viewer
-ipython
-pytest
-flake8
+Flask==1.1.2
+requests==2.25.0
+gunicorn==20.0.4
+DateTime==4.3
```
Comment (xofbd) [inline requirements.txt]: `datetime` is part of the standard library so I don't think we need to specify it.

**Review by xofbd — CHANGES_REQUESTED** [review]
Looks good; see my comment.

**Comment by xofbd** [comment]
@turbatuandrei Is this ready to review again? What happened when you removed `datetime` from `requirements.txt`

**Comment by turbatuandrei** [comment]
I removed the version for datetime

**Comment by xofbd** [comment]
@turbatuandrei But did `datetime` need to have been specified at all?

**Comment by turbatuandrei** [comment]
> @turbatuandrei Is this ready to review again? What happened when you removed `datetime` from `requirements.txt`

I was getting errors on running the app 
image

**Comment by xofbd** [comment]
@turbatuandrei  I don't seem to get an error? Was the error presenting on Travis CI?

File: requirements.txt (hunk 1 above)
Comment (turbatuandrei) [reply requirements.txt]: > @turbatuandrei I don't seem to get an error? Was the error presenting on Travis CI?

You are right, I removed it again and I'm not getting any errors in heroku logs

**Comment by turbatuandrei** [comment]
> @turbatuandrei I don't seem to get an error? Was the error presenting on Travis CI?

You are right, I removed it again and I'm not getting any errors in heroku logs :)

**Comment by turbatuandrei** [comment]
> > @turbatuandrei I don't seem to get an error? Was the error presenting on Travis CI?
> 
> You are right, I removed it again and I'm not getting any errors in heroku logs :)

No, unit tests were passing

**Review by xofbd — APPROVED** [review]

**Pull request merged**